### ISSUE TITLE
last.fm integration

### ISSUE THREAD

**Issue by not-ivy** [leinelissen/jellyfin-audio-player#214]
thank you for creating this app! i love the feature where you can download the songs and stream offline, i dont think its in the jellyfin app yet :p

i know there is a jellyfin plugin that integrates with last.fm, but that is server-side only. is it possible to do it on the client side? because i believe when you stream songs that are downloaded offline, the status isn't sent to the server, correct me if i am wrong 😅

**Comment by not-ivy** [comment]
nevermind, i just discovered the playback reporting option, and looks like it is enabled by default :p

still, would it be out of scope to create a client side integration?

**Comment by leinelissen** [comment]
Not necessarily, but I would strongly prefer to have Jellyfin as the sole back-end for the application. Is there a reason why the server-side plugin is insufficient?

**Comment by not-ivy** [comment]
in some rare cases if playback reporting fail(server is down while streaming locally, or a network issue), client will be able to handle the tracking

**Comment by magandrez** [comment]
@leinelissen FYI

**Comment by leinelissen** [comment]
Hey @magandrez, please create a new issue for this. Can you double-check that the listening events are coming in correctly in your admin dashboard? They should be available at https://[JELLYFIN-SERVER-URL]/web/#/dashboard/activity?useractivity=true
